plugins/modules/df_service.py

```diff
@@ -35,17 +36,17 @@
 options:
   env_crn:
     description: 
-        - The CRN of the CDP Environment to host the Dataflow Service
-        - Required when state is present
+      - The CRN of the CDP Environment to host the Dataflow Service
+      - The environment name can also be provided, instead of the CRN
+      - Required when state is present
     type: str
-    required: False
+    required: Conditional
     aliases:
       - name
-      - crn
   df_crn:
     description: 
-        - The CRN of the DataFlow Service, if available
-        - Required when state is absent
+      - The CRN of the DataFlow Service, if available
+      - Required when state is absent
     type: str
     required: Conditional
```

Comment:
Same as above -- Boolean

Reply:
See my comment above. I don't think Boolean cuts it.

Reply:
Fixed